src/evrythng/entities/products.py

```diff
@@ -59,3 +59,9 @@ def delete_product(product_id, api_key=None):
     assertions.datatype_str('product_id', product_id)
     url = '/products/{}'.format(product_id)
     return utils.request('DELETE', url, api_key=api_key)
+
+def read_all_actions(product_id, action_type='all', api_key=None):
```

Comment:
- Will you add a docstring for this and the other read_all_actions?
- I recommend adding in the parameters too.  Reference create_collection for guidelines.
- Perhaps action_type default should be None and you can use 'all' when None is provided?